apps/crunchy-postgres/workload/test.yml

```diff
@@ -0,0 +1,152 @@
+            replace: "{{ loadgen_lkey }}: {{ loadgen_lvalue }}"
+
+        - name: Replace namespace placeholder in Crunchy-loadgen
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "name_space"
+            replace: "{{ namespace }}"
+
+        - name: Replace service placeholder in Crunchy-loadgen
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "servicename"
+            replace: "{{ service_name.stdout }}"
+
+        - name: Replace database placeholder
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "dbname"
+            replace: "{{ lookup('env','DATABASE_NAME') }}"
+
+        - name: Replace password placeholder in Crunchy-loadgen
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "database_password"
+            replace: "{{ lookup('env','PASSWORD') }}"
+
+        - name: Replace database-user placeholder in Crunchy-loadgen
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "dbuser"
+            replace: "{{ lookup('env','DATABASE_USER') }}"
+
+        - name: Replace port placeholder in Crunchy-loadgen
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "port"
+            replace: "{{ lookup('env','PORT') }}" 
+
+        - name: Replace transaction placeholder in Crunchy-loadgen
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "database_transaction"
+            replace: "{{ lookup('env','TRANSACTIONS') }}"
+
+        - name: Replace parallel-transaction placeholder in Crunchy-loadgen
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "paralleltransaction"
+            replace: "{{ lookup('env','PARALLEL_TRANSACTION') }}"           
+    
+
+        - name: Create test specific namespace.
+          shell: kubectl create ns {{ namespace }}
+          args:
+           executable: /bin/bash
+          when: namespace != 'litmus'
+
+        - name: Checking the status  of test specific namespace.
+          shell: kubectl get ns {{ namespace }} -o jsonpath='{.status.phase}'
+          args:
+           executable: /bin/bash
+          register: npstatus
+          until: "'Active' in npstatus.stdout"
+          delay: 30
+          retries: 10
+        
+        - name: Create Crunchy-postgres Loadgen Job
+          shell: kubectl apply -f {{ crunchy_loadgen }} -n {{ namespace }} 
+
+        - name: Verify loadgen pod is running 
+          shell: kubectl get pods -n {{ namespace }} -l {{ loadgen_label }} -o jsonpath='{.items[0].status.phase}'
+          args:
+            executable: /bin/bash
+          register: result
+          until: "'Running' in result.stdout"
+          delay: 30
+          retries: 15
+
+        - name: Verifying Load-generation
+          shell: kubectl exec -it pgset-0 -n {{ namespace }} -- psql -c '\dt'
+          register: output
```

Comment:
@shashank855 can we get the pod name from task instead of giving name directly???

Reply:
@nsathyaseelan I have made the changes.